src/less/less/mixins-common.less

```diff
@@ -1,41 +1,5 @@
 @import "variables-common.less";
```

Comment:
Looks like we can remove this import now.

Reply:
Yeah, to be honest, you'll probably see a bunch of redundant imports like this. Another of those things where we'll need to sweep through all files at some point and get consistency of our headers/imports.